app/views/patients/_patient_dashboard.html.erb

```diff
@@ -1,77 +1,12 @@
-                     label: t('patient.dashboard.weeks_along'),
-                     autocomplete: 'off',
-                     help: t('patient.dashboard.currently', weeks: patient.last_menstrual_period_now_weeks, days: patient.last_menstrual_period_now_days) %>
-      </div>
-
-      <div class="col mt-4">
-        <%= f.select :last_menstrual_period_days,
-                     options_for_select(days_options, patient.last_menstrual_period_days),
-                     autocomplete: 'off',
-                     skip_label: true,
-                     help: t('patient.dashboard.called_on', date: patient.initial_call_date.strftime("%m/%d/%Y")) %>
-        <%= f.label :last_menstrual_period_days, t('common.days_along'), class: "sr-only" %>
-      </div>          
-
-      <div class="col-3">
-        <%= f.date_field :appointment_date,
-                         label: t('patient.shared.appt_date'),
-                         autocomplete: 'off',
-                         help: t('patient.dashboard.approx_gestation', weeks: patient.last_menstrual_period_at_appt_weeks, days: patient.last_menstrual_period_at_appt_days) %>
-      </div>
-    </div>
-
-    <div class="row">
-      <div class="col-4">
-        <%= f.text_field :primary_phone,
-                         value: patient.primary_phone_display,
-                         label: t('patient.dashboard.phone'),
-                         autocomplete: 'off' %>
-      </div>
-
-      <div class="col">
-        <%= f.text_field :pronouns,
-                         value: patient.pronouns,
-                         autocomplete: 'off' %>
-      </div>
-
-      <div class="col">
-        <div class="form-group">
-          <label for="status"><%= t 'patient.shared.status' %> <%= tooltip_shell status_help_text(patient) %></label>
-          <input type="text" value="<%= patient.status %>" class="form-control form-control-plaintext" id="patient_status_display" autocomplete="off" disabled>
-        </div>
-      </div>
-
-      <div class="col-3">
-        <% if current_user.admin? %>
-          <div class="form-group">
-            <label for="admin-delete"><%= t 'patient.dashboard.delete_label' %></label>
-            <div>
-              <%= link_to t('patient.dashboard.delete'),
-                          patient_path(patient),
-                          class: 'btn btn-danger',
-                          method: :delete,
-                          data: { confirm: t('patient.dashboard.confirm_del', name: patient.name) } %>
-            </div>
-          </div>
-        <% end %>
-      </div>
-    </div>
-  <% end %>
-</div>
+  <%= render ReactComponent.new("PatientDashboardForm", raw_props: { 
+    patient: patient.as_json, 
+    weeksOptions: weeks_options.map { |opt| { label: opt[0], value: opt[1] } },
+    daysOptions: days_options.map { |opt| { label: opt[0], value: opt[1] } },
+    initialCallDate: patient.initial_call_date.strftime("%m/%d/%Y"),
+    statusHelpText: status_help_text(patient),
```

Comment:
at some point we'll need to find a way to re-rack this from changes (easiest example on main: setting an appointment date makes the status `Fundraising`, removing appointment dates makes the status `Needs Appointment`. That said I'm perfectly fine letting this regress a bit and merging this, and tackling this later, because I'd like to start running this react code in prod again. (Please make an issue if we do that so we don't forget.)

Reply:
I'm not totally following your comment but I think that's b/c I'm not as familiar with all the parts of the app as you are. Can you clarify by what you mean by "this"? The status help text? Or the status input itself?

Reply:
yeah, the status help text -- the content changes based on the state of the patient (and I believe that's a model field right now). On prod now it refreshes after a change without a page reload.

Reply:
cool cool, I went ahead and fixed that in this branch 🤗 hopefully no regressions now